### PR TITLE
Add SPACE ID - name resolution snap

### LINKED ISSUE #812: [New Snap] Space ID
## Checklist

All items in the list below needs to be satisfied.

- [x] Is the Snap repository publicly accessible and linked in this ticket? https://github.com/Space-ID/web3-name-snap
- [x] Is the Snap distributed on [npm](https://www.npmjs.com/) and linked in this ticket? https://www.npmjs.com/package/@web3-name-sdk/snap
- [x] Has an audit been performed and the audit report attached or linked in this issue? https://github.com/Space-ID/web3-name-snap/blob/main/docs/SlowMist%20Audit%20Report%20-%20web3-name-snap.pdf
- [x] Is a complete list of discovered vulnerabilities from the audit documented in this issue?
- [ ] For vulnerabilities that have been deemed necessary to be addressed, are the links to the fixes attached to this issue?
- [ ] For vulnerabilities that have been deemed not necessary to be addressed, is a reason for each of them documented in this issue?
- [ ] The corresponding pull request in this repo has been merged.

### PR DESCRIPTION
**Pull request by Montoya**
Closes #812 

Requires: 
- [ ] : #814

### CODE REVIEW

File: src/registry.json [hunk 1]
```diff
@@ -3567,6 +3567,29 @@
         }
       }
     },
+    "npm:@web3-name-sdk/snap": {
+      "id": "npm:@web3-name-sdk/snap",
+      "metadata": {
+        "name": "SPACE ID Web3 Domain ",
```
Comment (Mrtenz) [inline src/registry.json:3573]: There's one too many SPACEs here. 😅

```suggestion
        "name": "SPACE ID Web3 Domain",
```

**Review by Mrtenz — CHANGES_REQUESTED** [review]

**Comment by Montoya** [comment]
Confirmed to work with test case `hello.arb` on Ethereum mainnet 
hello_arb

**Review by FrederikBolding — APPROVED** [review]

File: src/registry.json [hunk 2]
```diff
@@ -3597,6 +3597,37 @@
...
+        "description": "After installing the SPACE ID Web3 Domain Snap, when transferring crypto, you can type in any web3 domains names you registered on SPACE ID directly instead of copying & pasting complex wallet addresses.",
+        "audits": [
+          {
+            "auditor": "SlowMist",
+            "report": "https://github.com/slowmist/Knowledge-Base/blob/master/open-report-V2/blockchain-application/SlowMist%20Audit%20Report%20-%20web3-name-snap_en-us.pdf"
+          }
+        ],
+        "category": "name resolution",
+        "support": {
+          "contact": "https://discord.com/channels/953120695431331901/978354253838049371",
```
Comment (Mrtenz) [inline src/registry.json:3619]: This doesn't work.

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/registry.json (hunk 2 above)
Comment (FrederikBolding) [reply src/registry.json:3619]: Good catch!
```suggestion
          "contact": "https://discord.gg/spaceid",
```
Wondering if we can just do this, this is linked in the docs.

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**